content/en/synthetics/ci.md

```diff
@@ -223,7 +278,61 @@ The configuration of synthetics test occurs in `**/*.synthetics.json` (configura
 }
 ```
 
-#### Running tests
```

Comment:
Would be nice to clarify that all these fields are optional and that the "original" configuration from the test will be used if they're missing.